open-sphere-base/core/src/main/java/io/opensphere/core/quantify/Quantify.java

```diff
@@ -0,0 +1,84 @@
+package io.opensphere.core.quantify;
+
+import io.opensphere.core.Toolbox;
+
+public class Quantify
```

Comment:
Documentation needed here for Quantify class

Reply:
fixed